hedera-node/src/test/java/com/hedera/services/state/initialization/ViewBuilderTest.java

```diff
@@ -63,18 +67,31 @@ public static void assertIsTheExpectedUtao(FCOneToManyRelation<EntityId, MerkleU
 		expected.getKeySet().forEach(key -> assertEquals(expected.getList(key), actual.getList(key)));
 	}
 
+	public static void assertIsTheExpectedTUtao(FCOneToManyRelation<EntityId, MerkleUniqueTokenId> actual) {
+		final var uniqId = new MerkleUniqueTokenId(tokenId, 2);
+		final var expected = new FCOneToManyRelation<EntityId, MerkleUniqueTokenId>();
+		expected.associate(tokenId, uniqId);
+		assertEquals(actual.getKeySet(), actual.getKeySet());
+		expected.getKeySet().forEach(key -> assertEquals(expected.getList(key), actual.getList(key)));
+	}
+
 	@Test
 	void rebuildOwnershipsAndAssociationsWorks() {
 		// given:
 		final var actualUta = new FCOneToManyRelation<EntityId, MerkleUniqueTokenId>();
 		final var actualUtao = new FCOneToManyRelation<EntityId, MerkleUniqueTokenId>();
+		final var actualTUtao = new FCOneToManyRelation<EntityId, MerkleUniqueTokenId>();
+
+		final var tokenStore = mock(TokenStore.class);
+		given(tokenStore.isTreasuryForToken(any(), any())).willReturn(false);
 
 		// when:
-		ViewBuilder.rebuildUniqueTokenViews(someUniqueTokens(), actualUta, actualUtao);
+		ViewBuilder.rebuildUniqueTokenViews(tokenStore, someUniqueTokens(), actualUta, actualUtao, actualTUtao);
 
 		// then:
 		assertIsTheExpectedUta(actualUta);
 		assertIsTheExpectedUtao(actualUtao);
+//		assertIsTheExpectedTUtao(actualTUtao); TODO case with ownership treasury
```

Comment:
What is the reason behind commenting this line

Reply:
The unit test setup is not completed for this actualTUtao